packages/pwa-kit-build/src/configs/webpack/config.js

```diff
@@ -244,7 +253,8 @@ const clientOptional = baseConfig('web')
                 ...optional('worker', './worker/main.js'),
                 ...optional('core-polyfill', resolve(projectDir, 'node_modules', 'core-js')),
                 ...optional('fetch-polyfill', resolve(projectDir, 'node_modules', 'whatwg-fetch'))
-            }
+            },
+            plugins: [analyzeBundle && getBundleAnalyzerPlugin('client-optional')].filter(Boolean)
```

Comment:
Please make sure that we extend from existing plugins:
`plugins: [...config.plugins, analyzeBundle && getBundleAnalyzerPlugin('client-optional')].filter(Boolean)`